### PR TITLE
Spanish Translation Grammar Fixes and Added Translations

### PR DESCRIPTION
**Pull request by rafa-gould**

### CODE REVIEW

**Comment by C47D** [comment]
Que tal @rafa-gould, pudiste haber mandado estos commits al otro pull-request que cerraste (#1851, #1848).

Saludos

**Comment by rafa-gould** [comment]
Solo inspecciona los cambios en 'es.po' y no los otros. Hay cambios ejecutados por "make translate", pero solo hice cambios en 'es.po' y para que sea más fácil ver las diferencias en los cambios antes de "make translate". Muchas gracias. 

--

Just check the 'es.po' changes and not the other ones. There are changes that were run by "make translate" but I only made changes in 'es.po' and to make it easier you can look at the differences in the changes before "make translate". Thank you so much.

File: locale/es.po [hunk 1]
```diff
@@ -326,14 +326,14 @@ msgstr "Brightness debe estar entro 0 y 255"
 
 #: shared-bindings/displayio/Display.c
 msgid "Brightness not adjustable"
-msgstr ""
+msgstr "Brightness no adjustable"
```
Comment (sabas1080) [inline locale/es.po]: Brillo no ajustable

**Comment by tannewt** [comment]
Thank you all! @erikriver the other files are ok because they were updated by `make translate` which changes file locations and timestamps.

**Pull request merged**